lib/utils.js

```
@@ -1,18 +1,34 @@
 'use strict';
 
 const convert = require('koa-convert');
-const is = require('is-type-of');
 const co = require('co');
 
+function isGeneratorFunction(obj) {
+  return obj
+    && obj.constructor
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)